tests/test_client.py

```diff
@@ -237,4 +237,52 @@ def test_bulk_get_latest(index_client):
     docs_null_excluded = index_client.bulk_get_latest(dids, skip_null=True)
     docs_null_included = index_client.bulk_get_latest(dids, skip_null=False)
     assert {doc.did for doc in docs_null_excluded} == new_dids
-    assert {doc.did for doc in docs_null_included} == null_dids
\ No newline at end of file
+    assert {doc.did for doc in docs_null_included} == null_dids
+
+
+@pytest.mark.parametrize("exclude, expectation", [
+    ("01.txt", 4),
+    ("test_bucket", 0),
+])
+def test_query_urls__exclude(indexd_loader, indexd_client, exclude, expectation):
+    indexd_loader("tests/data/documents.json")
```

Comment:
I think this is going to fail if one runs this tests from a folder other than the root, which is a pretty common thing I do personally. The `pkg_resources` module helps guarantee loading is always succeeds like absolute urls does

Reply:
Yeah I removed it because it was causing failures in travis https://app.travis-ci.com/github/NCI-GDC/indexclient/jobs/524530525

Reply:
That's a weird situation there. pkg_resources thinks the `tests` module is in indexd. Try adding an `__init__.py` inside the tests directory. That should resolve this folder first before looking into the `src` folder

Reply:
Sounds good. Looks like that did the trick